### PR TITLE
Advanced search

### PR DESCRIPTION
**Pull request by ebrelsford**
This PR add advanced search functionality:

image

Since users can select multiple tags at once it can be frustrating when using AND logic (a dataset must have all tags) so we flipped the tags to use OR logic via [a plugin](https://github.com/DataShades/ckanext-or_facet/). This requires a few small changes to the `.env` which we'll highlight inline.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -75,14 +75,14 @@ NGINX_PORT=80
 NGINX_SSLPORT=443
 
 # Extensions
-CKAN__PLUGINS="envvars image_view text_view datatables_view recline_view datastore datapusher natcap spatial_metadata spatial_query stats activity scheming_datasets"
+CKAN__PLUGINS="or_facet envvars image_view text_view datatables_view recline_view datastore datapusher natcap spatial_metadata spatial_query stats activity scheming_datasets"
```
Comment (ebrelsford) [inline .env.example:78]: Add `or_facet`

File: .env.example [hunk 2]
```diff
@@ -75,14 +75,14 @@ NGINX_PORT=80
...
 CKAN__HARVEST__MQ__TYPE=redis
 CKAN__HARVEST__MQ__HOSTNAME=redis
 CKAN__HARVEST__MQ__PORT=6379
 CKAN__HARVEST__MQ__REDIS_DB=1
 
 # Support searching by arbitrary polygon, not just by bounding box.
-CKAN__SEARCH__SOLR_ALLOWED_QUERY_PARSERS="field"
+CKAN__SEARCH__SOLR_ALLOWED_QUERY_PARSERS="field edismax bool"
```
Comment (ebrelsford) [inline .env.example:85]: Update SOLR config for `or_facet`.

File: .env.example [hunk 3]
```diff
@@ -109,3 +109,6 @@ CKAN__AUTH__PUBLIC_USER_DETAILS=false
 
 # Add ckanext.scheming schemas here
 CKAN___SCHEMING__DATASET_SCHEMAS="ckanext.natcap:natcap_dataset_schema.yml"
+
+# Config for or_facet, allow OR logic for facets
+CKANEXT__OR_FACET__OPTIONAL="extras_sources_res_formats extras_placenames tags"
```
Comment (ebrelsford) [inline .env.example:114]: Add fields to use OR for with `or_facet`.

**Comment by ebrelsford** [comment]
@phargogh This should be set now!

**Review by phargogh — APPROVED** [review]

**Pull request merged**